src/components/Ratings/docs/RatingsExtendedExample.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import { Ratings, Mode } from '../index';
+import extendedStyles from './RatingsExtendedExample.st.css';
+
+export const RatingsExtendedExample = props => (
+  <div>
+    <div>
+      <h3>Ratings Input</h3>
+      <Ratings
+        value={0}
+        onSelect={val => console.log(val)}
```

Comment:
In order to make this example more clear for consumers, let's add a state that saves this `Ratings`' value, and change it on `onSelect`.
I think you can use React hooks if you don't want to create a class for this.